### PR TITLE
Add SentimentAnalysis unit test

### PR DESCRIPTION
**Pull request by frankfliu**
## Description ##

Brief description of what this PR is about

- If this change is a backward incompatible change, why must this change be made?
- Interesting edge cases to note here

### CODE REVIEW

File: examples/src/test/java/ai/djl/examples/inference/SentimentAnalysisTest.java [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+import ai.djl.translate.TranslateException;
+import java.io.IOException;
+import org.testng.Assert;
+import org.testng.SkipException;
+import org.testng.annotations.Test;
+
+public class SentimentAnalysisTest {
+
+    @Test
+    public void testPoseEstimation() throws ModelException, TranslateException, IOException {
```
Comment (zachgk) [inline examples/src/test/java/ai/djl/examples/inference/SentimentAnalysisTest.java:27]: not pose estimation

Comment (frankfliu) [reply examples/src/test/java/ai/djl/examples/inference/SentimentAnalysisTest.java]: done

File: examples/src/main/java/ai/djl/examples/inference/SentimentAnalysis.java [hunk 2]
```diff
@@ -56,6 +57,8 @@ public static Classifications predict()
                 Criteria.builder()
                         .optApplication(Application.NLP.SENTIMENT_ANALYSIS)
                         .setTypes(String.class, Classifications.class)
+                        // This model was traced on CPU and can only run on CPU
```
Comment (stu1130) [inline examples/src/main/java/ai/djl/examples/inference/SentimentAnalysis.java:60]: I though it is because RNN bug  in MXNet MKLDNN

**Review by stu1130 — APPROVED** [review]

File: examples/src/main/java/ai/djl/examples/inference/SentimentAnalysis.java (hunk 2 above)
Comment (frankfliu) [reply examples/src/main/java/ai/djl/examples/inference/SentimentAnalysis.java:60]: This is a PT model.
